### ISSUE TITLE
Clicking Stop button puts the camera in software trigger mode

### ISSUE THREAD

**Issue by omarjamil** [UniExeterRSE/NetClamp#115]

**Comment by felicityguest** [comment]
I believe `triggersource` should be set to `internal` when the `Stop` button is pressed. At least that is what `stop_experiment()` currently does. @omarjamil, please correct me if I'm wrong.

**Comment by omarjamil** [comment]
That is correct. It should be put into `internal` mode as that is the mode for capturing live images.
